spec/bolt_server/transport_app_spec.rb

```diff
@@ -38,288 +38,255 @@ def file_data(file)
     expect(last_response.status).to eq(200)
   end
 
-  context 'with ssh target', ssh: true do
-    let(:target) { conn_info('ssh') }
-    let(:path) { '/ssh/run_task' }
-    let(:echo_task) {
-      {
-        'name': 'sample::echo',
-        'metadata': {
-          'description': 'Echo a message',
-          'parameters': { 'message': 'Default message' }
-        },
-        files: [{
-          filename: "echo.sh",
-          sha256: "foo",
-          uri: { path: 'foo', params: { environment: 'foo' } }
-        }]
-      }
-    }
-
-    it 'errors if both password and private-key-content are present' do
-      body = {
-        'task': echo_task,
-        'target': {
-          'password': 'foo',
-          'private-key-content': 'content'
-        },
-        'parameters': { "message": "Hello!" }
-      }
-
-      post path, JSON.generate(body), 'CONTENT_TYPE' => 'text/json'
+  context 'when raising errors' do
```

Comment:
I moved these app-wide test cases further up, above the transport-specific and action-specific tests. I also removed the task body content passed to this 404 test case, since they should never reach validation anyway .